lib/cisco_node_utils/cmd_ref/vxlan_vtep.yaml

```diff
@@ -0,0 +1,26 @@
+# vxlan_vtep
+---
+all_interfaces:
+  multiple:
+  config_get: 'show running-config interface all'
+  config_get_token: '/^interface\s+(nve.*)$/'
+
+mac_distribution:
+  config_get: 'show running-config interface all'
+  config_get_token: ['/^interface <name>$/', '/^host-reachability protocol (\S+)/']
+  config_set: ['interface <name>', '<no_cmd> host-reachability protocol <proto>', 'end']
```

Comment:
I'd prefer to use <state> here instead of <no_cmd>, which is somewhat nxos-centric (ie. XR tends to use 'disable'). That would also make it consistent with most of our other providers.